bot/reddit.py

```diff
@@ -56,6 +73,4 @@ async def reddit_flair_user(text, belt):
 
 def find_username(text):
```

Comment:
given we're going for a bit fancy regex  with a big fancy comment, we should move this to utils.py

Reply:
moved into utils